gatsby/src/docs/guides/alert-notifications/creating-custom-rules.mdx

```diff
@@ -0,0 +1,118 @@
+When it comes to being notified on your errors, there's a wide range of options between notifying all your project team members on each and every error instance and notifying them only once when an _Issue_ is first seen. Ideally, when an error happens, you want the right people to know about it in real time. For that, create custom **Alert Rules** in your project settings and define **who** to notify, about **what** error, **when** and **how**, making sure that errors are getting the right attention by the relevant team members.
+
+## Alert Rules
+
+Alert Rules enable powerful rule creation to notify developers of inbound errors through any channel or tool your team may be using. Alert rules are configured per project under `[Project Settings] > Alerts > Rules` tab where you’ll see a list of all active rules and can add new rules or modify existing ones. 
+
+Alert rules consist of:
+
+- A set of self-explanatory `Conditions` connected in an _ALL_, _ANY_, or _NONE_ relation.
+- The `Environment` where your code is deployed and running in which you'd like the alert rule applied.
+- The `Actions` that should be taken when the associated conditions are met. Mainly -- where you'd like to channel this alert.
+
+#### **Creating a good set of alert rules means:**
```

Comment:
guessing we're bolding these for styling? technically we shouldn't need to do that when its a header